pkg/cmd/roachtest/tests/failover.go

```diff
@@ -22,11 +23,43 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/registry"
 	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/spec"
 	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test"
+	"github.com/cockroachdb/cockroach/pkg/roachpb"
 	"github.com/cockroachdb/cockroach/pkg/roachprod/install"
 	"github.com/cockroachdb/cockroach/pkg/util/randutil"
+	"github.com/cockroachdb/errors"
 	"github.com/stretchr/testify/require"
 )
 
+var rangeLeaseRenewalDuration time.Duration
+
+func init() {
```

Comment:
Can we use the "func initializer" pattern (name I just made up, maybe there's a more official term), instead of a package-level `init`:

```go
var rangeLeaseRenewalDuration = func() time.Duration {
  // ...
}()
```

Reply:
Good idea, thanks. Done.